README.md

```diff
@@ -66,3 +66,68 @@ It should be noted that specifying a plan file on the command line has precedenc
 ```
 ./run_tests.py -c ../config.ini -p openstack
 ```
+
+# run-tests-podman
+
+This section covers building a framework container and some specifics related to running tests using containerized framework. This approach is useful when there is a need to run all the tests from within a locked-down network. It implies that container is built from a RHEL8 machine which has all the containerization prerequisites installed. Later, container file can be exported and imported to any machine within a locked-down network capable of running containers. 
+
+## Building a container image
+
+```
+podman build .
+``` 
+
+Build process installs all the required packages and copies the entire framework folder to the container image /testing folder.
+
+## Exporting the image
+
+```
+podman save <image_id> > image_filename.tar
+```
+This will export previously built image to a file so it can be tranported and used on any other machine.
+
+## Importing the image 
+
+```
+podman load < image_filename.tar
+```
+This will import the image on the any machine where tests are going to be run.
+
+## Running the tests
+
+```
+podman run -v </path/to/folder/containing/needed/files>:/testing/external:z -e openstackrc=<openstackrc_file> -e plan=<plan>  <image_id>
+``` 
+The command will mount local foder containing ansible test repository and all other needed files, to container folder /testing/external. It will take openstackrc file reference and plan reference defined in telco-tests plans folder.
+ 
+Mandatory files and folders:
+- <test_repository>
+- <config_file>
+- <extravars_file>
+- <openstackrc_file> 
+
+Keep in mind when adjusting config.ini that container is not aware of any files outside /testing/external.
+
+Here is an example snippet of config.ini references having in mind container folder /testing/external:
+```
+[General]
+test_directory = /testing/external/telco-tests
+plans_directory = /testing/external/telco-tests/plans
+inventory = /testing/external/telco-tests/inventory/hosts
+report    = /testing/external/report.csv
+extra_vars = /testing/external/extravars
+iterations = 1
+max_failures = 3
+output_directory = /testing/external/output
+```
```

Comment:
Instead of just providing an example, could we actually include a file in the container?
potentially with some -e for iterations etc?

Reply:
If file is within a container and not in external folder, it is baked in. No changes to the file can be done without entering container, changing file and committing running container. User can easily change files in external and iterate correspondingly.

Reply:
We decided to keep it for now and eventually change it when we have config.ini defaults in place.